app/Livewire/Source/Github/Change.php

```
@@ -148,47 +148,48 @@ public function checkPermissions()
+
+                return;
+            }
+
+            // Verify the private key exists and is accessible
+            if (! $this->github_app->privateKey) {
+                $this->dispatch('error', 'Private Key not found. Please select a valid private key.');
+
+                return;
+            }
+
             GithubAppPermissionJob::dispatchSync($this->github_app);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Jobs\GithubAppPermissionJob' in method 'checkPermissions'. (undefined)

(StaticAccess)